lib/pushData.js

```diff
@@ -42,23 +44,19 @@ function assemblePush(push) {
     };
 }
 
-function markSent(id) {
+export async function markSent(id) {
     const now = moment.tz('Europe/Berlin').format();
-    return request.patch({
-        uri: urls.push(id),
-        json: true,
-        body: { delivered: true, 'delivered_date': now },
-        headers: { Authorization: 'Token ' + process.env.CMS_API_TOKEN },
-    }).then((response) => {
+
+    try {
+        const response = await request.patch({
+            uri: urls.push(id),
+            json: true,
+            body: { delivered: true, 'delivered_date': now },
+            headers: { Authorization: 'Token ' + process.env.CMS_API_TOKEN },
+        });
         console.log(`Updated push ${id} to delivered`, response);
-    }).catch((error) => {
-        console.log(`Failed to update push ${id} to delivered`, error);
-        throw error;
-    });
+    } catch (e) {
+        console.log(`Failed to update push ${id} to delivered`, e.message);
+        throw e;
+    }
 }
-
-module.exports = {
-    getLatestPush,
-    assemblePush,
-    markSent,
-};
```

Comment:
This should not be removed but become the default export to not change behaviour.